eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -571,44 +571,6 @@ public function testLoadContentInfoThrowsNotFoundException()
         /* END: Use Case */
     }
 
-    /**
-     * Test for the loadContentInfoList() method.
-     *
-     * @see \eZ\Publish\API\Repository\ContentService::loadContentInfoList()
-     */
-    public function testLoadContentInfoList()
```

Comment:
Please restore all the tests that were dropped, so we can be sure there are no breaking changes :D